homeassistant/components/systemmonitor/sensor.py

```diff
@@ -47,7 +48,7 @@
     ],
     "ipv4_address": ["IPv4 address", "", "mdi:server-network", None, True],
     "ipv6_address": ["IPv6 address", "", "mdi:server-network", None, True],
-    "last_boot": ["Last boot", "", "mdi:clock", "timestamp", False],
+    "last_boot": ["Last boot", None, "mdi:clock", DEVICE_CLASS_TIMESTAMP, False],
```

Comment:
Note that we don't change the icon, even though it should not be set when device class is set. I want to avoid a breaking change in this PR. We can fix this separately.

Reply:
Although technically setting the unit to None is also a breaking change.

Reply:
Lets add a small note so it ends up in the release notes. Better to mention it, even if it is small.